distro_logo_licensing.md

```diff
@@ -0,0 +1,147 @@
+# Licensing and Credits for Upstream Artwork
+## WIP  This is DRAFT WORKING MATERIAL AND NOT TO BE RELEASED
+
+The artwork for various Operating Systems and Distributions is used or combined under the following licenses or terms:
+
+## GPL
+## MIT
+## Apache
+## Berkeley License
+
+### openbsd
+https://www.openbsd.org/art4.html
+https://www.openbsd.org/policy.html
+
+
+
+## Specific Permissions
+
+## Specific Permissions to be sought
+
+### Haiku
+
+ - [ ] [Haiku](https://www.haiku-inc.org/trademarks/allowed_usage/)
+
+ https://www.haiku-inc.org/trademarks/usage_requirements/
+
+ “Haiku® and the HAIKU logo® are registered trademarks of Haiku, Inc. and are developed by the Haiku Project”.
+ “Haiku, Inc.” shall be a hyperlink to “http://www.haiku-inc.org”
+“Haiku Project” shall be a hyperlink to “http://www.haiku-os.org”
+
+
+### alma
+ - [ ] [Alma Linux](https://almalinux.org/p/the-almalinux-os-trademark-usage-policy/ )
+   - Problematic:
+"You may not change any logo except to scale it. This means you may not add elements to the logo, change the colors or proportions of the logo, distort the logo, or combine the logo with other logos."
```

Comment:
We could play it safe and remove the quickemu logo overlay.

Reply:
It would be the safe option.  I would hope they (and most) would respond positively to an approach for specific permission as not only is this good promotion/inclusion of their project(s) but your work is beautifully and respectfully done. As a long-time DBA the one that worries me most is a trademarked O ;-)